EpsilonCCS/Inc/MpptRtr.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "stm32f4xx.h"
+#include "stm32f4xx_hal_can.h"
+
+#define MPPT_HEARTBEAT_CAN_FREQ 200
+#define MPPT_STDID 0x600U
+#define MPPT_DLC 8
+
+extern CAN_HandleTypeDef hcan1;
+extern osPoolId canTxPool;
+extern osMessageQId canTxQueue;
+
+typedef struct // Datasheet is here: https://drive.google.com/open?id=0B4Qf5FVsKl-SVDRFY3RoWUt0RVU
+{
+    uint32_t stdId;
+    uint32_t channel; //not sure of the data type, but it is a binary encoded number from 0-3.
+} MpptCanMsg;
+
+void mpptRtrTask(void const* arg);
+void sendCanTask(void const* arg);
```

Comment:
Sending CAN is not directly coupled with the MPPT logic, we can also potentially need to send CAN messages that are not MPPT related, so it'd be better to have this in a separate file especially because the naming and usage of this function is so generic.

Reply:
Should I keep the mppt can message a separate structure from the regular CAN messages? I would say that it has a more unique thing with the channel value.